nextflow-parsing/NextflowParsingFunction/pom.xml

```diff
@@ -98,7 +98,7 @@
     <dependency>
       <groupId>io.dockstore</groupId>
       <artifactId>openapi-java-client</artifactId>
-      <version>1.12.0-alpha.1-SNAPSHOT</version>
+      <version>1.13.0-SNAPSHOT</version>
```

Comment:
If this was originally tested with only a snapshot, this should probably be pinned at 1.12.2. 
Snapshots in Maven are intended to be ephemeral (i.e. can go away) and it can also change over time (which I realize now is the opposite of how we use the name in Dockstore (oops)